com.github.mimillieuh.thinkercad-desktop.yml

```diff
@@ -0,0 +1,47 @@
+app-id: com.github.mimillieuh.thinkercad-desktop
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: com.github.mimillieuh.thinkercad-desktop
+rename-desktop-file: ThinkerCAD.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it really support wayland out of the box? If yes then line 12 should be fallback-x11. Otherwise remove line 13.